src/pages/Preferences.vue

```diff
@@ -57,6 +58,9 @@ export default Vue.extend({
         'preferences/fetchPreferences',
         this.credentials
       );
+    },
+    reloadPreferences() {
+      ipcRenderer.send('preferences-reload');
```

Comment:
Hmm, `window.reload()` from here isn't enough? (I guess it doesn't really matter either way)

Consider updating `electron-ipc.d.ts`, even though it's currently broken (#1961).

Reply:
I think I perceive it to be more consistent to handle window lifecycle events via passing messages to the main process, but I don't think that matters too much here.. we don't have any mechanism for refreshing outside of the renderer.